x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/dashboards/actions/unlink_dashboard.tsx

```diff
@@ -20,10 +20,12 @@ export function UnlinkDashboard({
   currentDashboard,
   onRefresh,
   assetType,
+  canDeleteCustomDashboard,
 }: {
   currentDashboard: DashboardItemWithTitle;
   onRefresh: () => void;
   assetType: InfraCustomDashboardAssetType;
+  canDeleteCustomDashboard: boolean;
```

Comment:
I think you could drop the `CustomDashboard` in the property name, you're this component is about dashboard.
```suggestion
  canDelete: boolean;
```

Reply:
renamed ✅